src/pages/invitee/EventInvite.jsx

```diff
@@ -1,17 +1,28 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Navbar from "../../components/Navbar";
 import Footer from "../../components/Footer";
 import { useNavigate, useParams } from "react-router-dom/dist";
+import moment from "moment/moment";
+import fetchApi from "../../services/fetchApi";
+import { BASE_URL, EVENT_BY_TOKEN, ADD_PARTICIPANTS } from "../../services/rootEndPoints";
```

Comment:
All these are implemented in the userServices.js file. only your call in the userServices.js should be imported 👍🏽. kindly check how it is being used in the signin, dashboard, create event, and forgot password page